ansible/provision-and-execute-tests/playbook.yml

```diff
@@ -161,48 +181,65 @@
     - name: Install Node JS
       ansible.windows.win_command: "nvm install {{ node_version }}"
 
-    - name: Run NPM version
-      ansible.windows.win_shell: "nvm use {{ node_version }} ; npm -v"
-      register: npm_version_out
+    - name: Copy test suite
+      ansible.builtin.copy:
+        src: ../../integration-tests/test-suite
+        dest: 'C:\Windows\Temp\'
+
+    - name: Install test-suite dependencies
+      ansible.windows.win_shell: "nvm use {{ node_version }} ; npm i"
+      args:
+        chdir: '{{ test_suite_folder }}'
 
-    - name: NPM version result (to be removed)
-      debug:
-        var: npm_version_out
+    - name: Remove previous reports
+      ansible.windows.win_file:
+        path: '{{ test_suite_folder }}\reports'
+        state: absent
 
-    - name: Run Node version
-      ansible.windows.win_shell: "nvm use {{ node_version }} ; node -v"
-      register: node_version_out
+    - name: Run test-suite
+      ansible.windows.win_shell: "nvm use {{ node_version }} ; npm run test"
+      args:
+        chdir: '{{ test_suite_folder }}'
+        creates: '{{ test_suite_folder }}\reports'
+      environment:
+        LOGGING_ENDPOINT: https://staging-log-api.newrelic.com/log/v1
+        NERD_GRAPH_URL: https://staging-api.newrelic.com/graphql
+        API_KEY: "{{ new_relic_api_key }}"
+        ACCOUNT_ID: "{{ new_relic_account_id }}"
+        MONITORED_FILE: '{{ monitored_file }}'
+#        MONITORED_TCP_PORT: "{{ monitored_tcp_port }}" # waiting for https://github.com/fluent/fluent-bit/issues/6671
```

Comment:
Hey but the issue is already merged, isn't it?

Reply:
Also, this makes me think: the FB version we're using in the version YML files for Windows is also 2.0.8, I think. We should probably go back to 1.9.7 (or whatever version we are currently using). But this might cause that the binary is named `td-agent-bit`, so that would be extra work for us that we'd need to undo anyway later. Why not directly upgrade to the latest version available?

OK, here's my vote: let's leave it to 2.0.8 for now and in a next PR we try to upgrade _all of them_ to the latest version available.